packages/common/core-utils/package.json

```diff
@@ -42,7 +54,7 @@
 		"test:coverage": "c8 npm test",
 		"test:mocha": "mocha",
 		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
-		"tsc": "tsc",
+		"tsc": "tsc-multi --config ../../../common/build/build-common/tsc-multi.cjs.json",
 		"typetests:gen": "fluid-type-test-generator",
```

Comment:
It feels like if we are going to bother with tsc-multi for cjs, then we should also rename the .d.ts files (in another change).